pkg/alerts/prometheus.go

```diff
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"github.com/prometheus/client_golang/api"
-	"github.com/prometheus/client_golang/api/prometheus/v1"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
```

Comment:
why?

Reply:
That was automatically done by go, so I didn't fight it.
It's terrible, IMO. Want me to remove it?